catalyst/dl/registry.py

```diff
@@ -14,8 +19,39 @@ def _callbacks_loader(r: Registry):
 CALLBACKS.late_add(_callbacks_loader)
 Callback = CALLBACKS.add
 
+
+def _transforms_loader(r: Registry):
+    try:
+        import albumentations as m
+        r.add_from_module(m)
+    except ImportError as ex:
+        logger.warning(
+            "albumentations not available, to install albumentations, "
+            "run `pip install albumentations`."
+        )
+        if os.environ.get("USE_ALBUMENTATIONS", "0") == "1":
+            raise ex
+
+
+TRANSFORMS = Registry("transform")
```

Comment:
maybe, `"transforms"`?

Reply:
I think, no
we use `"transforms"` to define a list of transforms inside transform (e.g. for `albumentations.Compose`) and as registry provides factory by name it will lead to key duplication, e.g.:
```
transforms: Compose
transforms:
  - transform: Normalize
  - transform: ToTensorV2
```

Reply:
thinking, 
`registry.transform(MyTransform)` vs `registry.transforms(MyTransform)`